core-web/libs/template-builder/src/lib/components/template-builder/components/add-style-classes-dialog/add-style-classes-dialog.component.html

```diff
@@ -8,8 +8,8 @@
         [multiple]="true"
         [dropdown]="isJsonClasses$ | async"
         [autofocus]="true"
-        (onKeyUp)="onKeyUp($event)"
         (completeMethod)="filterClasses($event)"
+        dotSelectItem
```

Comment:
Do we have a way to test that we are passing this?

Reply:
Not sure, the only way is to use a selector that looks for this directive in the `spectator.query`